packages/next/src/server/app-render/create-component-tree.tsx

```diff
@@ -580,6 +597,7 @@ async function createComponentTreeInternal({
       </React.Fragment>,
       parallelRouteCacheNodeSeedData,
       loadingData,
+      true,
```

Comment:
Should this be false? I'm not 100% clear on what this branch is doing but it seems like this necessarily creates a response with holes given it's gated on forceDynamic

Reply:
The value should be `true` if the response may contain dynamic holes. Since this is a force-dynamic branch during static generation, it will contain dynamic holes.

Reply:
Woops, yeah, I flipped the meaning of the flag in my head when I wrote this comment.